### PR TITLE
Update TFE migration docs to use remote backend, finally

### PR DESCRIPTION
**Pull request by nfagerlund**
at long last 😭

### CODE REVIEW

File: content/source/docs/enterprise/migrate/workspaces.html.md [hunk 1]
```diff
@@ -59,94 +55,86 @@ In your shell, go to the directory with the Terraform configuration you're migra
...
-At this point, it's a good idea to use a checklist of workspaces to keep track of your progress.
+  If you answer "yes", Terraform will migrate all states. If you answer
+  "no", Terraform will abort.
 
-## Step 11: Run `terraform init` and Answer "No"
+  Enter a value:
+```
 
-Run `terraform init`.
+Answer "yes."
```
Comment (mildwonkey) [inline content/source/docs/enterprise/migrate/workspaces.html.md]:
```suggestion
Answer "yes".
```

File: content/source/docs/enterprise/migrate/workspaces.html.md [hunk 2]
```diff
@@ -59,94 +55,86 @@ In your shell, go to the directory with the Terraform configuration you're migra
...
 
-When asked if you want to migrate state, **answer "no."**
+Some backends, including the default `local` backend, allow a special `default` workspace that doesn't have a specific name. If you previously used a combination of named workspaces and the special `default` workspace, the prompt will next ask you to choose a new name for the `default` workspace, since TFE doesn't support unnamed workspaces:
```
Comment (mildwonkey) [inline content/source/docs/enterprise/migrate/workspaces.html.md]: Do we have any conventions/suggestions/best practices for the alternate `default` namespace? (This is more about my own curiosity and not a request to make a change).

Comment (nfagerlund) [reply content/source/docs/enterprise/migrate/workspaces.html.md]: TBH, I don't think we know enough about how people are using alternate workspaces in CLI terraform. Part of why the default case was blowing up on migration for so long is that no one here thought it made any sense to keep using `default` once you had any named workspaces. 

If I think in terms of the order things would happen in... probably you'd start a new configuration on default, tweaking and adjusting some new infrastructure you're not using yet. Then you'd get it the way you liked it and start using it as production. Then you'd want to spin up an alternate instance of it to test something dangerous, so you'd make a new workspace. In that scenario, `default` would be prod. But I don't know about other scenarios. I have to just assume that if you're using alternate workspaces, you already had to think it through enough that you probably know what your default workspace is used for.

File: content/source/docs/enterprise/migrate/workspaces.html.md [hunk 3]
```diff
@@ -59,94 +55,86 @@ In your shell, go to the directory with the Terraform configuration you're migra
...
-
-- **"Do you want to copy only your current workspace?"** Answer "yes."
+- Use the `remote` backend.
+- In the `organization` attribute, specify the name of your TFE organization.
+- The `hostname` attribute is only necessary with private TFE instances. You can omit it if you're using the SaaS version of TFE.
+- In the `prefix` attribute in the `workspaces` block, specify a prefix to use when naming newly created TFE workspaces. You'll use short workspace names on the CLI, and long workspace names with the prefix added in TFE's UI and API. See [the remote backend documentation][remote-backend] for details.
 
-    Terraform init can only handle the default workspace, so you'll use a different command for the others.
-- **"Do you want to copy existing state to the new backend?"** Answer "yes."
+## Step 6: Run `terraform init`, Answer "Yes", and Name the Default Workspace
```
[the remark this answers is not in the document]
Comment (nfagerlund) [reply content/source/docs/enterprise/migrate/workspaces.html.md]: Sure!

**Comment by nfagerlund** [comment]
@mildwonkey Thanks for the review! 🙌🏼

**Pull request merged**